src/components/Button/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import Button from 'react-bootstrap/Button';
+import style from './index.module.scss';
+
+/*
+    To use this component, pass the following props:
+    
+    > buttonLabel : Pass a string value for the button label.
+    
+    > isLarge : Pass true if you want to have a large button. 
+                otherwise it will return default button.
+
+    > isSmall : Pass true if you want to have a small button. 
+                otherwise it will return default button.
+    
+    > isSmall : Pass true if you want to have a outline button. 
```

Comment:
it is better for the size to be controllable via string. for example:
size="lg" - large size
size="sm" - small size
size="def" - default size